lms/envs/common.py

```diff
@@ -2967,4 +2967,7 @@
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
 #####RACCOONGANG########
 DOC_LINK_BASE_URL = ''
+
+# JIVOSITE widget settings:
+JIVOSITE_WIDGET_ID = os.environ.get('JIVOSITE_WIDGET_ID')
```

Comment:
But, from devops talk I was sure there is one true way to get some settings-param: yaml > json > env > settings > code. So, we should discuss it...